packages/babel-traverse/src/path/removal.js

```diff
@@ -7,7 +7,9 @@ export function remove() {
   this._assertUnremoved();
 
   this.resync();
-  this._removeFromScope();
+  if (this.opts && !this.opts.noScope) {
```

Comment:
When is `this.opts` undefined?

Reply:
I don't know ... CI reports an error. I didn't find out exactly what caused it. I also feel very confused, but in order not to affect compatibility, I added this judgment.
[CI Link](https://travis-ci.com/babel/babel/jobs/286667571)

Reply:
The check should be `!this.opts || !this.opts.noScope`. If `opts` is not defined, `noScope` is not true so the scope is created.

Reply:
oh no! You're right.

Reply:
I stumbled upon [this](https://github.com/babel/babel/blob/f544753bb8c9c7a470d98e897b089fd31b83d1f6/packages/babel-traverse/src/path/context.js#L114-L119) just now...